### PR TITLE
Use env variables to set SMTP settings

### PR DESCRIPTION
**Pull request by jarifibrahim**
Fixes https://github.com/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io/issues/204

Note - The application will return an Internal Server Error if any of these variables are not set.

### CODE REVIEW

**Comment by jarifibrahim** [comment]
Closing this PR since we do not need email related features at the moment.

https://github.com/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io/pull/224 makes this PR obsolete

File: oshc/oshc/settings.py [hunk 1]
```diff
@@ -159,11 +159,12 @@
...
-EMAIL_HOST = 'smtp.gmail.com'
-EMAIL_HOST_USER = 'email'
-EMAIL_HOST_PASSWORD = 'password'
-EMAIL_PORT = 587
+EMAIL_HOST = os.getenv('EMAIL_HOST', '')
+EMAIL_HOST_USER = os.getenv('EMAIL_ID', '')
+EMAIL_HOST_PASSWORD = os.getenv('EMAIL_PASSWORD', '')
+EMAIL_PORT = os.getenv('EMAIL_PORT', '')
 EMAIL_USE_TLS = True
+
```
Comment (tapaswenipathak) [inline oshc/oshc/settings.py:171]: Register/Login is of no good use if we don't confirm the email. Currently we have it hidden on master, once we are up with check we would make it live. This is needed and cannot be removed. Did you get the chance to set it on Heroku? or does this not work for you? looks good.

Comment (jarifibrahim) [reply oshc/oshc/settings.py:171]: @tapasweni-pathak There is a known issue with Gmail and Heroku. Heroku servers are located all around the world. When we try to send an email via Heroku server, Gmail blocks the request considering the unusual origin of the request. We could try and see if this works, but I'm not sure if this `email verification via emails from gmail` should be used on production.

Comment (tapaswenipathak) [reply oshc/oshc/settings.py:171]: This happens with mutt too if you use it with Gmail, [this](https://support.google.com/accounts/answer/6010255?hl=en) needs to be set up in that case and we can restrict to heroku. Can you please drop a blog or something, as I'm aware about [this](https://blog.heroku.com/tech_sending_email_with_gmail) but not about the known issue, thanks!

**Comment by jarifibrahim** [comment]
@tapasweni-pathak Let's merge this and see if it works. We'll never know until we try ;)

**Pull request merged**